src/status_im/ui/screens/notifications_center/styles.cljs

```diff
@@ -68,9 +69,11 @@
    :left     16})
 
 (def title-text
-  {:margin-left  72
-   :margin-top   12
-   :margin-right 50})
+  (let [title-text-width (* @(re-frame/subscribe [:dimensions/window-width]) 0.65)]
```

Comment:
its better to pass width is a parameter from the view

Reply:
I agree! 
working on this